ccc/parameters.py

```diff
@@ -274,7 +272,7 @@ def read_json_revision(obj):
         a valid JSON text.
 
         '''
-        return taxcalc.Parameters._read_json_revision(obj, 'revision')
+        return paramtools.Parameters.read_params(obj, 'revision')
```

Comment:
`Parameters.adjust` accepts JSON strings so you don't have to load them through `read_json_revision`--you could pass them directly to `Parameters.adjust()`. If you want to use `Parameters.read_params` like a static method, you can do it like this for now: 

```python
In [2]: paramtools.Parameters.read_params(None, '{"my": "json"}')                                                             
Out[2]: {'my': 'json'}
```

Out of curiosity, why is `read_json_revision` a static method? `read_params` doesn't use any instance data. So, I could make it a static method to fit your use case.